### PR TITLE
Update Guardfile

### PR DESCRIPTION
**Pull request by rsalevsky**
* The folder stucture changed in the last days so we need also
  adapt the Guardfile.
* Set vi:syntax for vim syntax highlighting

### CODE REVIEW

File: Guardfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env ruby
```
Comment (tboerger) [inline Guardfile]: But that's wrong, it's not an executable so this should not get a shebang. for enabling syntax highlighting there are other options.

**Comment by tboerger** [comment]
Except the shebang I'm fine with that.

**Comment by tboerger** [comment]
# vi:syntax=ruby

**Comment by MaximilianMeister** [comment]
:+1:

**Pull request merged**